### PR TITLE
Made minor modifications to the old content

### PR DESCRIPTION
**Pull request by sabeena21**

### CODE REVIEW

**Comment by journey2w** [comment]
@sabeena21 
As I asked you guys, please write strange requests. If you have a question, please add a new comment after pull request.

Please use pull request template and update it when you send a PR.

```
### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #1

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: SRMDEACR-21

~                        

```

File: docs/application/dotnet/api/overview.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+Tizen .NET provides a rich set of interfaces allowing you to build compelling TV, mobile, and wearable (preview) applications which achieve native performance. The programming environment includes the following:
 
-- **.NET Standard API**, which implements the .NET Base Class library, allows you to use the well known C# language base class libraries and features, such as collections, threading, file I/O, and LINQ.
-- **Xamarin.Forms**, which allows you to efficiently build a user interface from standard components in C# or XAML.
-- **TizenFX API**,
-   which allows you to access platform-specific features not covered by the generic .NET and Xamarin.Forms features,  such as system information  and status, battery status, sensor date, and account and connectivity services.  For the full specification, see the <a href="tizenfx/index.html" target="api">TizenFX API Reference</a>
+- **.NET Standard API** implements the .NET Base Class library, and allows you to use the well-known C# language base class libraries and features, such as collections, threading, file input/output (I/O), and Language Integrated Query (LINQ).
```
Comment (journey2w) [inline docs/application/dotnet/api/overview.md]: **.NET Standard API**, which implements the .NET Base Class library, allows you ... is better.

File: docs/application/dotnet/api/overview.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+- **Xamarin.Forms** allows you to efficiently build a user interface from standard components in C# or XAML.
+- **TizenFX API** allows you to access platform-specific features not covered by the generic .NET and Xamarin.Forms features, such as system information and status, battery status, sensor date, and account and connectivity services.  
+  For the full specification, see the [TizenFX API Reference]( https://github.com/jkpu/tizen-docs/blob/b49532622a70fd10dec59657fe9d531cea5ab6af/docs/application/dotnet/api/tizenfx/index.html).
```
Comment (journey2w) [inline docs/application/dotnet/api/overview.md]: We don't provide links to a personal repo.
Why did you change it?

File: docs/application/dotnet/api/overview.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
 
 ## .NET Standard API
 
-One of the major components of .NET Core is the .NET Standard. The .NET APIs provided by Tizen .NET follow the .NET Standard 2.0. The column titled "2.0" in the [official list](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) of supported CoreFX APIs lists all the available .NET APIs.
+One of the major components of .NET Core is the .NET Standard. The .NET APIs provided by Tizen .NET follows the .NET Standard 2.0. The column titled as 2.0 in the [official list](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) of the supported CoreFX APIs lists all the available .NET APIs.
```
Comment (journey2w) [inline docs/application/dotnet/api/overview.md]: **The .NET APIs** is the subject, so **follow** is correct.
How can I trust your review on?

File: docs/application/dotnet/api/overview.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 
 ## Xamarin.Forms
 
 [Xamarin.Forms](https://developer.xamarin.com/guides/xamarin-forms/getting-started/) provides cross-platform APIs, which allow you to create user interfaces that can be shared across platforms. The Tizen.NET Visual Studio extension enables Tizen support for Xamarin.Forms.
 
-You can efficiently build your Tizen .NET application UI and its supporting logic using the Xamarin.Forms APIs. Extended details for these APIs are available on the [Xamarin Web site](https://developer.xamarin.com/api/namespace/Xamarin.Forms/).
+You can efficiently build your Tizen .NET application user interface and its supporting logic using the Xamarin.Forms APIs. Extended details for these APIs are available on the [Xamarin Web site](https://developer.xamarin.com/api/namespace/Xamarin.Forms/).
```
Comment (journey2w) [inline docs/application/dotnet/api/overview.md]: UI is acceptable. **UI** instead of **user interface** is more used in technical documents.

File: docs/application/dotnet/api/overview.md [hunk 5]
```diff
@@ -28,25 +28,25 @@ You can efficiently add Tizen wearable device specific user interfaces. The exte
 
 ## TizenFX API
 
-The Tizen Platform-Specific API allows applications to call into platform-specific functionality from the shared code. This functionality enables Xamarin.Forms applications to do things a native application can do, without causing the portable part of the application to become littered with operating system-specific details.
+The Tizen platform-specific API allows applications to call into platform-specific functionality from the shared code. This functionality enables Xamarin.Forms applications to do things a native application can do, without causing the portable part of the application to become littered with operating system-specific details.
```
Comment (journey2w) [inline docs/application/dotnet/api/overview.md]: We used to "Tizen Platform-Specific API" before. However, The name of API was change to "TizenFX API".
Please use "The TizenFX API" instead of "The Tizen platform-specific API".

File: docs/application/dotnet/api/overview.md [hunk 6]
```diff
@@ -28,25 +28,25 @@ You can efficiently add Tizen wearable device specific user interfaces. The exte
...
 
-The following table specifies the API Level supported by each version of the Tizen platform.
+The following table specifies the API Level supported by each version of the Tizen platform:
 
 | Platform Version | API Level | Notes                                    |
 | ---------------- | --------- | ---------------------------------------- |
 | Tizen 4.0 M2     | 4         | [Platform Highlights](https://developer.tizen.org/tizen/tizen/tizen-4.0) |
 
-The following table lists the supported Tizen platform-specific API namespaces. For the full specification, see the <a href="tizenfx/index.html" target="api">TizenFX API Reference</a>
+The following table lists the supported Tizen platform-specific API namespaces. For the full specification, see the [TizenFX API Reference]( https://github.com/jkpu/tizen-docs/blob/b49532622a70fd10dec59657fe9d531cea5ab6af/docs/application/dotnet/api/tizenfx/index.html).
```
Comment (journey2w) [inline docs/application/dotnet/api/overview.md]: Same as above.

File: docs/application/dotnet/api/overview.md [hunk 7]
```diff
@@ -28,25 +28,25 @@ You can efficiently add Tizen wearable device specific user interfaces. The exte
...
-| Tizen.Account      | Provides CRUD (Create, Read, Update, Delete) account management functionality and the OAuth Core RFC 6749 protocol. |
+| Tizen.Account      | Provides Create, Read, Update, Delete (CRUD) account management functionality and the OAuth Core RFC 6749 protocol. |
 | Tizen.Applications | Provides the Tizen application framework, including application state change events, inter-application messaging, and notification services. |
 | Tizen.Common       | Provides predefined color names.         |
 | Tizen.Content      | Provides content management services, such as file and media downloads, storing and indexing audio and video content, and associating content types with helper applications. |
 | Tizen.Location     | Manages geographical location services and geofencing. |
-| Tizen.Maps         | Maps API provides a developer with a set of functions, helping to create Maps aware applications. |
-| Tizen.Messaging    | Provides functions to receive push notifications, email, and messages |
+| Tizen.Maps         | Maps API provides a developer with a set of functions, helping to create maps aware applications. |
```
Comment (journey2w) [inline docs/application/dotnet/api/overview.md]: Maps is the name of "Maps".

File: docs/application/dotnet/api/overview.md [hunk 8]
```diff
@@ -56,4 +56,4 @@ The following table lists the supported Tizen platform-specific API namespaces.
 | Tizen.System       | Provides device-specific services, including status, system information and settings, control of haptic devices and sensor control and access. |
 | Tizen.Telephony    | Provides call, modem, network, and SIM information. |
 | Tizen.Uix          | Provides functions to recognize the speech and to synthesize voice from text and playing synthesized sound data. |
-| Tizen.Webview      | Provides functions to access web pages and web contents. |
+| Tizen.Webview      | Provides functions to access webpages and web contents. |
```
Comment (journey2w) [inline docs/application/dotnet/api/overview.md]: Why did you change the words?

Comment (sabeena21) [reply docs/application/dotnet/api/overview.md]: @journey2w, according to our style guide we are directed to use "webpages" as one word.

Comment (journey2w) [reply docs/application/dotnet/api/overview.md]: There are lots of "a web page" and "web pages" in our guides. I didn't see your style guide as well as it is not standard that you change just one page.

**Comment by mijin85cho** [comment]
@sabeena21, to merge this PR, please resolve the conflict.

**Comment by sabeena21** [comment]
Since this PR is having merge conflict. I will be closing this PR.
Referencing with the new PR[].

**Comment by sabeena21** [comment]
@mijin85cho, I will be closing this PR and would raise a new PR to avoid merge conflict.
The new PR
